packages/teleport/src/Users/UserList/UserList.tsx

```diff
@@ -68,59 +67,54 @@ export default function UserList({
   }
 
   const data = sortAndFilter(searchValue);
-  const tableProps = { pageSize, data };
+  const tableProps = { pageSize, data, search: searchValue, onSearchChange };
```

Comment:
`searchValue` can be renamed to `search`

Reply:
leaving as is, there is going to be table update soon